src/plugins/recordTypes/collectionobject/optionLists.js

```diff
@@ -379,6 +380,10 @@ export default {
         id: 'option.recordStatuses.ready-for-review',
         defaultMessage: 'ready for review',
       },
+      'revision-needed': {
+        id: 'option.recordStatuses.revision-needed',
+        defaultMessage: 'revision needed',
+      },
       temporary: {
         id: 'option.recordStatuses.temporary',
         defaultMessage: 'temporary',
```

Comment:
Here it looks like we've added two values to the list, ready-for-review and revision-needed. So it's good that there are messages for them. Those messages will get merged in to the existing ones from cspace-ui. But we're also redefining messages for approved, in-process, new, and temporary. Those will also get merged into the existing messages, but they're exactly the same as the existing messages, so they have no effect. In general it's better to keep your plugin as small as possible -- everything should be additional config, or an override to existing config that makes a change. That makes it faster to download/parse/merge the plugin, and makes it easy to see exactly what the plugin does.